src/index.spec.ts

```diff
@@ -4,4 +4,17 @@ describe('configurations', () => {
     it('does not error when creating sdk object w/o token', () => {
         expect(() => briteRest()).not.toThrow();
     });
+
+    it('should compile while using async await', async () => {
```

Comment:
I'll be testing the base `request` function in my next PR so we'll indirectly be testing this. But it's cool to keep now. Can you put it in a different `describe` though?

Reply:
yeah of course!